els-back/app/Http/Controllers/Admin/CategoryController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\Category;
+use Illuminate\Http\Request;
+
+class CategoryController extends Controller
+{
+    public function index(Request $request)
+    {
+        $per_page = $request->query('per_page') ?? 10;
+
+        return Category::paginate($per_page);
+    }
+
+    public function store(Request $request)
+    {
+        return Category::create($request->toArray());
+    }
+
+    public function update(Request $request,Category $category)
+    {
+        return $category->update($request->toArray());
+    }
```

Comment:
Backend validation is more important than FE. Pls add validation. Atleast required validation